cpp/src/arrow/array/array_test.cc

```diff
@@ -534,7 +534,7 @@ static ScalarVector GetScalars() {
       std::make_shared<UInt64Scalar>(3),
       std::make_shared<DoubleScalar>(3.0),
       std::make_shared<Date32Scalar>(10),
-      std::make_shared<Date64Scalar>(11),
+      std::make_shared<Date64Scalar>(864000000),
```

Comment:
Is this literal meant to be an invalid value?
There is an extra 0 compared to `1000 x 60 x 60 x 24`.

Reply:
While I did not mean for it have an extra 0, it is still a valid value. So long as values for date64 are multiples of 86400000, they are valid (in this case the number just represents 10 whole days in milliseconds rather than 1 day).